### PR TITLE
zh-translation:content/zh/docs/tasks/observability/metrics/querying-metrics/index.md 

### PR DESCRIPTION
**Pull request by vivian99-wu**
ref: https://github.com/servicemesher/istio-official-translation/issues/1315

### CODE REVIEW

File: content/zh/docs/tasks/observability/metrics/querying-metrics/index.md [hunk 1]
```diff
@@ -1,119 +1,110 @@
...
-    browser or issue the following command:
+    以 Bookinfo 为例，在 web 浏览器中访问 `http://$GATEWAY_URL/productpage` 或执行如下命令：
 
     {{< text bash >}}
     $ curl http://$GATEWAY_URL/productpage
     {{< /text >}}
 
     {{< tip >}}
-    `$GATEWAY_URL` is the value set in the [Bookinfo](/docs/examples/bookinfo/) example.
+    `$GATEWAY_URL` 是在 [Bookinfo](/zh/docs/examples/bookinfo/) 应用中设置的值。
```
Comment (yuxiaobo96) [inline content/zh/docs/tasks/observability/metrics/querying-metrics/index.md]:
```suggestion
    `$GATEWAY_URL` 是在 [Bookinfo](/zh/docs/examples/bookinfo/) 示例中设置的值。
```

Comment (vivian99-wu) [reply content/zh/docs/tasks/observability/metrics/querying-metrics/index.md]: 感谢指正。

**Pull request merged**